sbom_generator.py

```diff
@@ -397,20 +400,26 @@ def cli_main():
             signer_id=signer_id,
             albs_url=albs_url,
         )
+        sbom_type = 'build'
     else:
         sbom = get_info_about_package(
             args.rpm_package_hash,
             signer_id=signer_id,
             albs_url=albs_url,
         )
-    # TODO: insert here formatter of SBOM and pass to it:
-    #       sbom
-    #       args.output_file
-    #       args.format_type (CycloneDX or SPDX)
-    #       args.format_mode (JSON or XML)
-    #
+        sbom_type = 'package'
+
     # TODO: remove it as debug line
-    logging.info(json.dumps(sbom, indent=4))
+    # logging.info(json.dumps(sbom, indent=4))
```

Comment:
May be we should get rid of this debug lines? or add some `verbose` argument that will set debug level for logging

Reply:
I will, I still need to integrate the latest changes from Stepan

Reply:
Fixed